R/PlainTextParam.R

```diff
@@ -367,19 +372,20 @@ setMethod("loadResults", signature(object = "MsBackendMzR",
 #' Chromatographic peaks
 #' @noRd
 .export_chrom_peaks <- function(x, path = character()) {
-    write.table(chromPeaks(x), file = file.path(path, "chrom_peaks.txt"))
-    write.table(as.data.frame(chromPeakData(x)),
+    write.table(chromPeaks(x), file = file.path(path, "chrom_peaks.txt"),
```

Comment:
I added `sep = "\t"` to all `write.table`/`read.table`. The default `sep = " "` might be problematic if there are white spaces in the file names or similar.

Reply:
Ah of course ! it was working  fine so I thought it was okay , but i should keep in mind that it's better to be safe than sorry :)